packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -32,13 +33,14 @@ const codeDetails: IFluidCodeDetails = {
 
 const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]]);
 
-function createLoader(testDriver: ITestDriver) {
+function createLoader(testDriver: ITestDriver, runId?: number) {
```

Comment:
i don't think runId should be optional

Reply:
It's also called from `initialize` in the orchestrator process which has no runId.